js/toolbar.js

```diff
@@ -17,7 +17,7 @@
 
 let WRAP = true;
 const $j = jQuery.noConflict();
-
+let variable=0;
```

Comment:
spaces around the = please.
maybe a better name for the variable that gives a hint as to what it is for?